### ISSUE TITLE
Training Time

### ISSUE THREAD

**Issue by AmingWu** [baker-laboratory/NA-MPNN#1]
Dear Authors,

How long does this method need to train?

**Comment by AndrewKubaney** [comment]
Hello!

The training details (GPU type, training hyperparameters, etc.) can be found in our preprint (https://www.biorxiv.org/content/10.1101/2025.10.03.679414v2). We trained both the design and specificity models for ~100k batches, which took ~2 days. However, based on validation set performance, we chose earlier checkpoints to release.
